### DISCUSSION TITLE
Dependency on Spyder

### DISCUSSION THREAD

**Discussion by cellsummer** [lifelib-dev/lifelib#22] [General]
Does lifelib rely on Spyder IDE? How do I open a model using a different python IDE such as vscode?

**Comment by fumitoh** [comment]
lifelib doesn't rely on Spyder. You can open a model using IPython in VS code like:

```python
>>> import modelx as mx

>>> m = mx.read_model(r"C:\Users\path\to\model")
```
GUI is available only with Spyder.
